### ISSUE TITLE
No SSIDs discovered under Kali 2021.3

### ISSUE THREAD

**Issue by plumped** [savio-code/fern-wifi-cracker#155]
Hi @savio-code  thank you for your great work!-
Anyway i got some trouble here with the searching results, in manual mode everything works great (also with wifite a.nd kismet)
My problem is that i dont get any results during scanning... Its forever "Active" and don't bring up some results here.. :(

Do you have any idea?
Kali Version 2021.3
Phyton Version 3.9.7
Aircrack Version rcrack-ng1.6 - (C)2
Qt Version 5.15.4

Thank you!

**Comment by absoda** [comment]
having same exact issue as plumped

**Comment by savio-code** [comment]
We are now working on the issue and will provide a fix before weekend

**Comment by WILDBIRDY92** [comment]
Helo Savio, I have exactly the same problem with Kali Version 2021.3
Do you have any solution ?
Thank's a lot by advance

**Comment by WILDBIRDY92** [comment]
Helo Savio,
Do you have any news concerning the fix ?
Thank's a lot

**Comment by savio-code** [comment]
Please run fern in command like this:
fern-wifi-cracker
Post any errors displayed on terminal here

**Comment by WILDBIRDY92** [comment]
Here it is the error message

┌──(kali㉿kali)-[~]
└─$ sudo fern-wifi-cracker             
[sudo] Mot de passe de kali : 
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
wash: aucun processus trouvé
Error occurred during scan:  'NoneType' object has no attribute 'find'

**Comment by WILDBIRDY92** [comment]
Hello Savio,
Do you have any news following my last message ?
Thank's a lot for your help

**Comment by WILDBIRDY92** [comment]
Hello Savio,
Just to inform you that I have tried with another wifi dongle and the problem is still present so, the problem don't come from the wifi adapter. => We get no results during the scan
Tjank's a lot for your help

**Comment by savio-code** [comment]
Last question, did you select the xterm option.

Sent from Yahoo Mail on Android 
 
  On Thu, 30 Sep 2021 at 6:10 pm, ***@***.***> wrote:   


Hello Savio,
Just to inform you that I have tried with another wifi dongle and the problem is still present so, the problem don't come from the wifi adapter. => We get no results during the scan
Tjank's a lot for your help


—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by WILDBIRDY92** [comment]
Even with xterm option selected, there is no scan result in wep and wpa menu ! nothing !
you can try it on your side

**Comment by WILDBIRDY92** [comment]
Hello Savio,
Do you have some news ?
Thank's a lot

**Comment by WILDBIRDY92** [comment]
Hello Savio
Still no news ?

**Comment by thanweer17** [comment]
> Even with xterm option selected, there is no scan result in wep and wpa menu ! nothing ! you can try it on your side

yes even for me the same problem. waiting for the patch. not getting any scan results even after 15 mins

**Comment by WILDBIRDY92** [comment]
Me too, waiting the patch since september !

**Comment by savio-code** [comment]
Is this error displayed everytime you try to scan?

Sent from Yahoo Mail on Android 
 
  On Sun, 7 Nov 2021 at 11:36 pm, ***@***.***> wrote:   


Me too, waiting the patch since september !


—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by WILDBIRDY92** [comment]
hello, yes, everytime ! the program isn't working
you have tried yourself ? you don't have the problem ?

**Comment by savio-code** [comment]
A fix will be made and uploaded next week..  Due to tight schedule, i havent had the time to fix the issue.

**Comment by Corywelds** [comment]
Any day now..

**Comment by defuuss** [comment]
Any News ?

**Comment by mohakawtar** [comment]
hello, the program isn't working

**Comment by mattyteee** [comment]
Kail 2021.4
Fern WiFi 3.0
Python 3.9.9
Aircrackrcrack-ng1.6- 9c0-2
Qt Version 5.15.6




sudo fern-wifi-cracker           
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find'
Error occurred during scan:  'NoneType' object has no attribute 'find

**Comment by vaskovass** [comment]
I am having the same issue. Using Kali via Live, persistence mode. The wireless adapter Alfa, works in Parrot OS. Tried to manually set it to monitor mode and it successfully scans for AP in the terminal. However, when trying to do it via Fern, the device switches to wlan0mon but not showing any APs. I just hear the fans speeding up, when I click the scan button and thats it.

**Comment by Hector6704** [comment]
Same problem here (kali 2021.4)... Did anyone try to use an older fern version?

**Comment by Hector6704** [comment]
@savio-code didn't you said that you were planning a fix?

**Comment by savio-code** [comment]
The issue has been fixed, update to version 3.14, Sorry for the long delay

**Comment by calidad-loca** [comment]
Having same problem - no AP's found Kali 2021.3 running in VirtualBox. I will try the update, but I do notice the Fern screen reads "No new update is Available" for 3.0
![image](https://user-images.githubusercontent.com/49651010/167933058-16befb80-5616-4af2-af18-017ae3817acb.png)
